test/suite/lighthouse/helpers.go

```diff
@@ -0,0 +1,230 @@
+package lighthouse
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/jenkins-x/bdd-jx/test/helpers"
+
+	"github.com/jenkins-x/bdd-jx/test/utils"
+	"github.com/jenkins-x/jx/pkg/gits"
+	"github.com/jenkins-x/jx/pkg/util"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+const (
+	defaultContext    = "pr-build"
+	lhQuickstart      = "golang-http"
+	brokenJenkinsXYml = `buildPack: go
+pipelineConfig:
+  pipelines:
+    overrides:
+      # Replace make-build on pullRequest in any stage/lifecycle with "exit 1" so the pipeline will fail.
+      - pipeline: pullRequest
+        name: make-linux
+        step:
+          sh: sleep 60 && exit 1
```

Comment:
why wait a minute here?

Reply:
I wanted some delay to make sure the builds don’t just fail instantly, but this probably a little overkill. I’ll drop it to 15 seconds - I think that should be fine.

Reply:
I think that is better. Things take long enough already